src/app/shared/task-provider/task-provider.component.html

```diff
@@ -39,13 +39,36 @@
                 </div>
             </div>
         </div>
+        <mat-toolbar class="small-toolbar">
+            Tracker Ticket Status Mapping
+        </mat-toolbar>
+        <div class="table-box padded-container" [formGroupName]="taskProviderConfigKey">
+            <div class="layout-row">
+                <mat-form-field class="flex">
+                    <input matInput 
+                    placeholder="Done (Comma Separated)" 
+                    [pattern]="commaSeparatedRegex" 
+                    [formControlName]="trackerTicketStatusMap.DONE"/>
+                    <mat-error *ngIf="doneStatusControl.errors && doneStatusControl.errors['pattern']">
+                        It should be a comma separated string
+                    </mat-error> 
+                    <mat-hint> 
+                        <mat-icon>info</mat-icon>
```

Comment:
Add CSS to beautify the info section, similar to the one done for provider custom fields.

Reply:
Done